fpdf/template.py

```diff
@@ -74,26 +74,37 @@ def load_elements(self, elements):
         self.elements = elements
         self.keys = [v["name"].lower() for v in self.elements]
 
+    def _parse_colorcode(self, s):
+        """ Allow hex and oct values for colors """
+        s = s.strip()
+        if not s:
+            raise ValueError('Foreground and Background must be numeric')
```

Comment:
maybe consider putting a different error message as s is "not empty" here, not "not numeric". (unless i am missing something?)

Reply:
I phrased it in this way to inform the user about what's supposed to go there. "Not empty" doesn't give that information.
The documentation could also be more specific about this, but fleshing that out is a project for another time...